src/App.js

```diff
@@ -90,55 +109,25 @@ export default function App() {
     }
   }
 
-  function handleOsc1Type(waverform1) {
-    setOsc1Type(waverform1);
-  }
-
-  function handleOsc2Type(waveform2) {
-    setOsc2Type(waveform2);
-  }
-
-  function handleOsc1FrequencyChange(oscillator1) {
-    setOsc1Frequency(oscillator1);
-  }
-
-  function handleOsc2FrequencyChange(oscillator2) {
-    setOsc2Frequency(oscillator2);
-  }
-
-  function handleFilterCutoffChange(cutOff) {
-    setFilterFrequency(cutOff);
-  }
-
-  function handleFilterResonanceChange(resonance) {
-    setResonance(resonance);
-  }
-
-  function handleFilterTypeChange(filterType) {
-    setFilterType(filterType);
+  function createChangePatch(key) {
```

Comment:
Nice usage of a higher-order function!

To deepen your knowledge: https://eloquentjavascript.net/05_higher_order.html